react/src/containers/ParkShow.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import ParkInfo from './ParkInfo'
+import ParkReviews from './ParkReviews'
+import FollowButton from '../components/FollowButton'
+
+class ParkShow extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      park: null
+    }
+  }
+
+  componentDidMount() {
+    //fetch call for 'park' state
+  }
+
+  render() {
+    let ratings
+    if(this.state.park) {
+      ratings = [
+        {name: "avg_rating",  value: this.state.park.avg_rating},
+        {name: "dog_friendly_avg_rating",  value: this.state.park.dog_friendly_avg_rating}
+      ]
+    }
+
+    return(
+      <div>
+      	<FollowButton />
+      	{this.state.park &&
+          <ParkInfo
+        	  ratings={ratings}
+        	/>
+        }
```

Comment:
this looks strange and hacky, you should not need to be doing this. Only be rendering the ParkInfo component please